client/app/stores/requests_in_flight_store.coffee

```diff
@@ -0,0 +1,35 @@
+Immutable = require 'immutable'
```

Comment:
@m4dz : this file name is generic but used only for discovering; are you sure you need this? cant you move this into `accountStore`, or `routerStore` instead?